activerecord/test/schema/mysql2_specific_schema.rb

```diff
@@ -40,6 +40,17 @@
 END
 SQL
 
+  ActiveRecord::Base.connection.execute <<-SQL
+DROP PROCEDURE IF EXISTS topics;
+SQL
+
+  ActiveRecord::Base.connection.execute <<-SQL
```

Comment:
We need to enable `CLIENT_MULTI_STATEMENTS` for multiple statements into a single command.
https://dev.mysql.com/doc/refman/5.6/en/c-api-multiple-queries.html

Should we enable the option for these queries?